applications/GeoMechanicsApplication/CMakeLists.txt

```diff
@@ -0,0 +1,167 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+message("**** configuring KratosGeoMechanicsApplication ****")
+
+# PYBIND11
+include(pybind11Tools)
+
+include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/StructuralMechanicsApplication )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/GeoMechanicsApplication )
+
+## Define application core sources (pure C++ parts)
+set( KRATOS_GEO_MECHANICS_APPLICATION_CORE_SOURCES
+
+    ## Application class and variables
+    ${CMAKE_CURRENT_SOURCE_DIR}/geo_mechanics_application.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/geo_mechanics_application_variables.cpp
+
+    # Utilities
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/structural_mechanics_element_utilities.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/static_condensation_utility.cpp
+
+    # Processes
+
+    # Constitutive Laws
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/bilinear_cohesive_3D_law.cpp
```

Comment:
if you check the template app in the Core you can see that we have simplified this a lot